browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1335,6 +1343,8 @@ void RewardsDOMHandler::SaveAdsSetting(const base::ListValue* args) {
     const auto is_enabled =
         value == "true" && ads_service_->IsSupportedLocale();
     ads_service_->SetEnabled(is_enabled);
+  } else if (key == kShouldShowPublisherAdsOnParticipatingSites) {
+    ads_service_->SetShowPublisherAdsOnPariticipatingSites(value == "true");
```

Comment:
Can we have a `kTrue` (or better named) const string for these?

Reply:
Fixed